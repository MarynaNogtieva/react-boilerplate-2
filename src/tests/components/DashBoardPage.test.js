import React from 'react';
import { shallow } from 'enzyme';
import DashBoardPage from '../../components/DashBoardPage';

test('should render Dashboard Page properly', () => {
  const wrapper = shallow(<DashBoardPage />);
  expect(wrapper).toMatchSnapshot();
});
