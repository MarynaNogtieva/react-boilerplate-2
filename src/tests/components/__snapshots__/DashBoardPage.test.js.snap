// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard Page properly 1`] = `
<div>
  Dashboard page
</div>
`;
